assets/tests/hooks/useDetour.test.ts

```diff
@@ -48,7 +45,8 @@ describe("useDetour", () => {
       return Promise.resolve(Ok(detourShape))
     })
 
-    const { result } = renderHook(useDetourWithFakeRoutePattern)
+    const originalRoute = originalRouteFactory.build()
+    const { result } = renderHook(() => useDetour(originalRoute))
```

Comment:
I think you're looking for this syntax?

```suggestion
    const { result } = renderHook(useDetour, {
      initialProps: originalRouteFactory.build(),
    })
```

Reply:
Omg!

Reply:
Addressed [here](https://github.com/mbta/skate/pull/2696/commits/6cc24892ad37299888cb0c28a64b9db5cc7a820b), but I think [this (next commit)](https://github.com/mbta/skate/pull/2696/commits/7fdb8c9931158b351b62daa225379e0db0afcb67) might be even better.